libraries/fof/controller/controller.php

```diff
@@ -3347,56 +3347,15 @@ protected function _csrfProtection()
 				break;
 		}
 
-		$hasToken = false;
-		$session  = JFactory::getSession();
+		$token = new \Joomla\Cms\Session\CsrfToken(JFactory::getSession());
```

Comment:
The version *is* already determined at this point, it is the current version. This piece of code will *never* be executed in an 1.5 or 2.5 environment.